pyproject.toml

```
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "dj-registry"
+version = "0.1.0"
+description = "A simple, easy access key-value registry for Django."
+authors = ["yychen <ctchen@gmail.com>"]
+license = "MIT License"
+
+[tool.poetry.dependencies]
+python = ">=3.8"
+Django = ">=3"
```

- Using the [caret requirements](https://python-poetry.org/docs/dependency-specification/#caret-requirements) or [tilde requirements](https://python-poetry.org/docs/dependency-specification/#tilde-requirements) is a better practice. (Try to find out the reason why it's better than inequality or exact requirements!)
- Let's lower the allowed Python version to 3.6 since it's an old Python version that is just being deprecated in late 2021.
- Django version is a bit complicated. We can handle it in the following PR.

```suggestion
python = "^3.6"
Django = "^3"
```